app/policies/api/shops_policy.rb

```diff
@@ -0,0 +1,19 @@
+class Api::ShopsPolicy < ApplicationPolicy
+  def update?
+    # Assuming that only the shop owner can update the shop information
+    if user.is_a?(User) && record.user_id == user.id
+      true
+    else
+      false
+    end
```

Comment:
The `Api::ShopsPolicy` is assuming that only the shop owner can update the shop information. Ensure that this assumption is correct and that the appropriate permissions are being checked.